packages/amplication-server/src/core/entity/constants.ts

```diff
@@ -136,42 +136,6 @@ export const DEFAULT_ENTITIES: EntityData[] = [
           maxLength: DEFAULT_SINGLE_LINE_TEXT_MAX_LENGTH,
         },
       },
-      {
-        dataType: EnumDataType.Username,
-        name: "username",
-        displayName: "Username",
-        description:
-          "An automatically created field of the username of the user",
-        unique: false,
-        required: true,
-        searchable: true,
-        properties: {
-          maxLength: DEFAULT_SINGLE_LINE_TEXT_MAX_LENGTH,
-        },
-      },
```

Comment:
Why do we need to remove the username field?

Reply:
@abrl91 When you don't work with auth you don't need userName (there is no login)